pyanaconda/bootloader.py

```diff
@@ -680,12 +683,16 @@ def set_stage1_device(self, devices):
                     self.stage1_device = device.parents[0]
                 else:
                     self.stage1_device = device
-
                 break
+            errors.extend(self.errors)
 
         if not self.stage1_device:
             self.reset()
-            raise BootLoaderError("failed to find a suitable stage1 device")
+            msg = "Failed to find a suitable stage1 device"
+            if errors:
+                msg = msg + ": " + "; ".join(errors)
+                self.errors = errors
+            raise BootLoaderError(msg)
```

Comment:
This (the exception msg) gets to the error detail dialog for autopartitiong case.